summary/summary.py

```diff
@@ -73,43 +73,9 @@ def to_fiatstr(msat: Millisatoshi):
                               int(msat) / 10**11 * plugin.fiat_per_btc)
 
 
-# This is part of pylightning, but its just merged,
```

Comment:
how long has the updated pylightning release been available for?

Reply:
It already was in the pyln-client version the requirements.txt suggested for this plugin anyway, but since we do not track plugin deps precisely, I put that in in the past just to be sure. The pyln-client `plugin.fiat_per_btc` was added to master with commit 0efd72d88 what is a bit more than one year ago.